src/pyvesync/helpers.py

```diff
@@ -38,6 +39,14 @@ def req_headers(manager) -> dict:
         }
         return headers
 
+    @staticmethod
```

Comment:
I'm glad you noticed, and that was intentional, to segment different PRs into smaller chunks.
There was a pattern naming the classes, like: req_header, req_header_bybass, req_body_base, req_body_details etc...
But then there was bypass_header that did not follow the existing naming pattern. I didn't want to reuse that, since I would rather change every ocurrence for the the supposed 'following the pattern" nomenclature, but changing every ocurrence would made that PR even bigger. so I had plans for doing a PR to refactor that, cleaning and making it common in a following separate PR. So it's more readable.

Reply:
Can we keep as is? Or do I need to change? Waiting for directions...